Flow.Launcher/ViewModel/ResultsViewModel.cs

```diff
@@ -166,7 +166,7 @@ private void UpdateResults(List<ResultViewModel> newResults, CancellationToken t
             switch (Visbility)
             {
                 case Visibility.Collapsed when Results.Count > 0:
-                    Margin = new Thickness { Top = 8 };
```

Comment:
This part gives the margin of the list list and query box. In the absence of a seperator (classic themes), vertical centering seems to be out of place, so it was removed. It can be added as the Margin of the theme's basis separator attribute, and in recent themes, it is displayed with the separator.